manifest.json

```diff
@@ -28,20 +28,20 @@
                 "name": "domain",
                 "type": "domain",
                 "ask": {
-                    "en": "Choose a domain name for LBCAlerte",
-                    "fr": "Choisissez un nom de domaine pour LBCAlerte"
+                    "en": "Choose a domain name for Cheky",
+                    "fr": "Choisissez un nom de domaine pour Cheky"
                 },
                 "example": "example.com"
             },
             {
                 "name": "path",
                 "type": "path",
                 "ask": {
-                    "en": "Choose a path for LBCAlerte",
-                    "fr": "Choisissez un chemin pour LBCAlerte"
+                    "en": "Choose a path for Cheky",
+                    "fr": "Choisissez un chemin pour Cheky"
                 },
-                "example": "/LBCAlerte",
-                "default": "/LBCAlerte"
+                "example": "/Cheky",
+                "default": "/Cheky"
```

Comment:
Here you should use lowercases in example and default (yes, another existing defect :smile:)